editors-processor/src/test/resources/org/gwtproject/editor/processor/test02/TestEditor02.java

```diff
@@ -0,0 +1,27 @@
+package org.gwtproject.editor.processor.test02;
+
+import org.gwtproject.editor.processor.common.EditorBeanWithInterface;
+import org.gwtproject.editor.processor.common.WidgetImplementsAsEditor;
+import org.gwtproject.editor.client.annotation.IsDriver;
+import org.gwtproject.editor.client.Editor;
+import org.gwtproject.editor.client.SimpleBeanEditorDriver;
+
+public class TestEditor02 implements Editor<EditorBeanWithInterface> {
+  
+  private Driver driver;
+  
+  @IsDriver
+  interface Driver
+      extends SimpleBeanEditorDriver<EditorBeanWithInterface, TestEditor02> {
+    
+  }
+  
+  WidgetImplementsAsEditor id;
+  WidgetImplementsAsEditor label;
+  
+  public TestEditor02() {
+    driver = new TestEditor02_Driver_Impl();
+    driver.initialize(this);
+  }
```

Comment:
question: is this code actually run, or just generated? if it is run, we want to also call edit and flush, and make sure that values make it into the editor, and changes  back out again

Reply:
This code is only generated. It does not run. And in this special case, it is only input for the generator.